cmd/application.go

```diff
@@ -0,0 +1,153 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/civo/civogo"
+	"github.com/civo/cli/config"
+	"github.com/civo/cli/utility"
+	"github.com/spf13/cobra"
+)
+
+var appCmd = &cobra.Command{
+	Use:     "app",
+	Aliases: []string{"apps, application, applications"},
+	Short:   "Manage Applications inside your Civo account",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		err := cmd.Help()
+		if err != nil {
+			return err
+		}
+		return errors.New("a valid subcommand is required")
+	},
+}
+
+var appDomainCmd = &cobra.Command{
+	Use:     "domain",
+	Aliases: []string{"domains"},
+	Short:   "Details of your application domains",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		err := cmd.Help()
+		if err != nil {
+			return err
+		}
+		return errors.New("a valid subcommand is required")
+	},
+}
+
+var appConfigCmd = &cobra.Command{
+	Use:     "config",
+	Aliases: []string{"conf"},
+	Short:   "Configure your application",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		err := cmd.Help()
+		if err != nil {
+			return err
+		}
+		return errors.New("a valid subcommand is required")
+	},
+}
+
+var appScaleCmd = &cobra.Command{
+	Use:     "scale",
+	Aliases: []string{"change", "modify", "upgrade"},
+	Example: "civo app scale APP-NAME PROCESS-NAME=PROCESS-COUNT",
+	Short:   "Scale processes of your application",
+	Args:    cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
Store the function in another file but invoke that function here. 

`Run : scaleCmd()`

Reply:
Done.